modules/filter/package.json

```diff
@@ -11,7 +11,7 @@
   "peerDependencies": {
     "react": "^17.0.0",
     "react-native": "^0.68.0",
-    "react-native-popover-view": "^4.0.0",
+    "react-native-popover-view": "^4.0.0 || ^5.0.0",
```

Comment:
?

Reply:
It defaults to expanding the scope of supported versions (sometimes??), so I think I need to configure it (??)